iati/about/templates/about/case_study_index_page.html

```diff
@@ -0,0 +1,68 @@
+{% extends "base.html" %}
```

Comment:
Comment for refactor - create an `about_base.html` which includes the side menu and featured event.